src/components/ImmutableDevices/ImmutableDevices.cy.js

```diff
@@ -166,8 +166,8 @@ describe('ImmutableDevices', () => {
 
     cy.get('td[data-label="Name"]')
       .first()
-      .find('.ins-composed-col > :nth-child(2) > a')
-      .trigger('click');
+      .find('.ins-composed-col > div > a')
+      .click();
```

Comment:
The `click()` was what fixed the failing Cypress test, which is odd, because I would expect `trigger('click')` to be the same and `click` would just be a shorthand.